packages/@okta/vuepress-site/docs/guides/configure-signon-policy/main/index.md

```diff
@@ -75,73 +76,107 @@ The following are step-by-step instructions to configure a global session policy
 
-The following are step-by-step instructions to configure another rule for the default authentication policy to prompt a user for an additional factor when the user is outside of the United States.
+### Create a passwordless policy
 
-> **Note:**  You can add as many rules to the default authentication policy that you want, but remember that the changes are applied to all new apps as it is a shared app policy.
+1. In the Admin Console, select **Security** > **Global Session Policy**.
 
-### Select the policy and add a rule
+1. [Create a policy container](#create-the-policy-container) and assign it to the **Full time employees** group.
 
-> **Note:** This example assumes that you've already [set up a Dynamic Zone](https://help.okta.com/okta_help.htm?id=ext_Security_Network). In this example, use a dynamic zone defined for IP addresses within the United States.
+1. Click **Create policy and add rule**. The Add Rule dialog appears.
 
-1. In the Admin Console, select **Security** > **Authentication Policies**.
+1. Enter a **Rule name**.
 
-2. Select **Default Policy** and then click **Add rule**. The Add Rule dialog appears.
+1. Configure IF conditions, which define the authentication context for the rule. For **IF User's IP is**, select **In zone**.
 
-3. Enter a **Rule name** such as **Prompt for an MFA factor when a user is outside the US**.
+1. Select the **Corporate Network** zone.
 
-4. Configure IF conditions to define the authentication context for the rule. Select **Not in any of the following zones** from the **AND User’s IP is** dropdown list.
+1. Leave the default AND conditions.
 
-    > **Note:** You can click the **Go to Network Zones** link to access the gateway settings that enable your choice of access. A [network zone](https://help.okta.com/okta_help.htm?id=ext_Security_Network) is a security perimeter used to limit or restrict access to a network based on a single IP address, one or more IP address ranges, or a list of geolocations. You can also create network zones using the [Zones API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/NetworkZone/).
+1. Configure THEN conditions, which define the authentication experience for the rule. For this use case, leave the default of **Allowed** for **THEN Access is**.
 
-5. In the text box, start typing the dynamic zone name and then select it when it appears in the list.
+1. Select that the session is established with the user entering **Any factor used to meet the Authentication Policy requirements**.
 
-6. Configure THEN conditions to define the authentication experience. For this use case, leave the default of **Allowed after successful authentication** for **THEN Access is**.
+1. Select that MFA is **Required** so that users in the **Full time employees** group are prompted for a secondary factor before they’re granted access.
```

Comment:
```suggestion
1. For **Multifactor authentication (MFA) is**, select **Required** so that users in the **Full time employees** group are prompted for a secondary factor before they’re granted access.
```